index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Mathias Wismann</title>
+</head>
+
+<body>
+    <header>
+        <div class="logo-container">
+            <h4>Welcome</h4>
+        </div>
+        <button type="button" id="hamburger"><img src="icon-img/incon-menu.png" alt=""></button>
+    </header>
+
+    <!-- Main content starts here -->
+    <main>
+        <section class="headline">
+            <div class="heading">
+                <h1>Hey there. I'm Mathias</h1>
+                <h2>I'm a software developer</h2>
+                <h3> I can help you build a product, feature or website. Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
+                    don't hesitate to contact me.</h3>
+            </div>
+
+            <ul>
+                <li class="media-link"><a href="https://twitter.com/mathias_wismann"><img src="icon-img/twitter.png"
+                            alt="twitter logo"></a></li>
+                <li class="media-link"><a
+                        href="https://www.linkedin.com/in/math%C3%ADas-wismann-bertolucci-8812a4194/"><img
+                            src="icon-img/linked-in.png" alt="linked-in logo"></a></li>
+                <li class="media-link"><a href="https://medium.com/"><img src="icon-img/medium.png"
+                            alt="medium logo"></a></li>
+                <li class="media-link"><a href="https://github.com/mwismann"><img src="icon-img/github.png"
+                            alt="github logo"></a></li>
+                <li class="media-link"><a href="https://angel.co/"><img src="icon-img/angellist.png"
+                            alt="angellist logo"></a>
```

Comment:
- [ ] Good job using a list :clap: , Kindly wrap the icons into anchor tags <a> as they are supposed to be links :recycle:

Reply:
Hi @GedeonTS ,

In this case, my icons (img element) are already wrapped in anchor tags:

`
<a href="https://github.com/mwismann"><img src="icon-img/github.png" alt="github logo"></a>
`

Were you referring to wrapping the list elements in anchor tags? 👀 
I'd appreciate it if you could clarify that doubt, thanks in advance 🙏